build-support/compile_time.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+#
+# Copyright 2021-present StarRocks, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Modified from hyrise project
+
+# The root directory that contains .git, i.e, the root directory of the project
+output=$(git rev-parse --show-toplevel)/compile_times.txt
```

Comment:
one more thing, what if the script is running under the workspace that is not a git repository, just a snapshot copy of the source code.

Reply:
Sounds reasonable